tests/vm_test.go

```diff
@@ -2122,7 +2123,13 @@ func stopVM(virtClient kubecli.KubevirtClient, vm *v1.VirtualMachine) *v1.Virtua
 }
 
 func retryWithMetadataIfModified(objectMeta metav1.ObjectMeta, do func(objectMeta metav1.ObjectMeta) error) (err error) {
-	return tests.RetryIfModified(func() error {
-		return do(objectMeta)
-	})
+	retries := 0
```

Comment:
I'm not convinced this function should exist even once, but this PR is replicating it. Can we avoid it completely? I think you had a former version of the code where the annotation was used of creation and retries were not needed.

Reply:
Yes, you're right. I'll do it in a new dedicated PR, this PR is only to remove `RetryIfModified` function from utils.

Reply:
I think it makes sense to do [this](https://github.com/kubevirt/kubevirt/pull/12031#discussion_r1624307557) here, shouldn't be a massive change

Reply:
@akalenyu Alright, I changed it. Let's see if it works, and I assume you meant to `retryIfModified` and not `retryWithMetadataIfModified`.